src/test/java/com/softserveinc/ita/SampleTest.java

```diff
@@ -1,12 +1,23 @@
-package com.softserveinc.ita;
+package com.softserveinc.ita.vsaroz;
```

Comment:
something weird with the package, why did you add here your package name, if this test file is in 'com.softserveinc.ita' package?